.cargo/config.toml

```diff
@@ -4,6 +4,7 @@ rustflags = [
     # internally, not by dependants.
     '--cfg=HYDROFLOW_GENERATE_DOCS',
 ]
+rustdocflags = [ '--cfg=docsrs', '--cfg=stageleft_runtime' ]
```

Comment:
Surprised that you need the `stageleft_runtime` one, it should be added by the `build.rs`?

Reply:
Seems it is not needed, removed